source/appModules/devenv.py

```diff
@@ -90,7 +93,16 @@ class AppModule(appModuleHandler.AppModule):
 	def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 		# Only use this overlay class if the top level automation object for the IDE can be retrieved,
 		# as it will not work otherwise.

```

Comment:
nit: As you moved the check for VsTextEditPaneClassName down, you should move this comment down there also.